User-Help/Scoring

```diff
@@ -0,0 +1,22 @@
+
+# Question scoring and ordering
+
+## Overview 
+
+Communities need a way to identify and reward quality contributions to their sites. They also need a way to sort post threads so higher quality content is shown first. Codidact solves this problem by assigning each post a score based on a ratio of upvotes to total votes, weighted to favor votes with fewer downvotes. For example, a post with seven upvotes will be shown higher than a post with nine upvotes and two downvotes. This method gives a slight boost to contributions that are consistently viewed positively over more controversial answers that gather both upvotes and downvotes. 
+
+## Calculations Used
+
+The actual score of a particular post is calculated as 
+![(upvote_count + z^2/2)/(upvote_count + downvote_count + z^2)](https://user-images.githubusercontent.com/10406565/78502558-9f804600-7759-11ea-966a-c9c0a37a4172.png)
```

Comment:
Can you add the images as files to the PR and use local links here?  We don't want our help to be dependent on third-party hosting, even GitHub.